### PR TITLE
add docs to readme

### PR DESCRIPTION
**Pull request by nichoth**
Add docs via this issue -- https://github.com/ssb-js/secret-stack/issues/69 .

### CODE REVIEW

**Review by staltz — CHANGES_REQUESTED** [review]
The paragraph above "This module:" already describes that secret-stack is built on secret-handshake and muxrpc, how about improving that paragraph instead of adding another one? And try to not explain the underlying modules, multiserver can explain itself in its README, muxrpc can explain itself in its README, etc.

**Review by nichoth — COMMENTED** [review]
+1 removed redundant parts

File: README.md [hunk 1]
```diff
@@ -2,11 +2,20 @@
...
-and [muxrpc](https://github.com/ssbc/muxrpc). This provides a framework
-to make building secure, decentralized systems easier.
+This provides a framework to make building secure, decentralized systems easier.
 (such as [ssb-server](https://github.com/ssbc/ssb-server) which this was refactored out of ;)
 
+This module:
+
+* uses [secret-handshake](https://github.com/auditdrivencrypto/secret-handshake) to set up a shared key for use between two peers with public / private keys (and verify the other peer)
+* uses [multiserver](https://github.com/ssb-js/multiserver) to handle different ways of connecting to other peers over different protocols (who you then handshake with)
+* uses muxrpc to provide a remote process call (rpc) interface to peers who have authenticated and connected allowing them to call methods on each other (like "please give me all mix's messages since yesterday"). This is all encrypted with the shared key set up by secret handshake.
```
Comment (staltz) [inline README.md:12]: Link to `muxrpc` here?

**Review by staltz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nichoth) [reply README.md]: good call I have added a link

Also I'm curious if these docs are accurate and helpful. It was just something I copied and pasted from a previous issue about docs

**Review by staltz — APPROVED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (staltz) [reply README.md]: Pretty accurate. The docs always look different anyway depending on who wrote them.

**Pull request merged**